packages/cli/src/contracts/LibDeployStub.sol

```diff
@@ -0,0 +1,23 @@
+// SPDX-License-Identifier: MIT
```

Comment:
Moving the "source of truth" of the LibDeploy.sol stub to its own file to avoid overriding it in during the deploy process.